apps/lib/s_cb.c

```diff
@@ -956,6 +956,9 @@ static int ssl_excert_prepend(SSL_EXCERT **pexc)
 {
     SSL_EXCERT *exc = app_malloc(sizeof(*exc), "prepend cert");
 
+     if (!exc) {
```

Comment:
False positive, `app_malloc()` doesn't return if the allocation fails.

Reply:
It would need to look at the link lines or build dependencies to figure out which sources were used.

We should fix the one in `test/testutil/apps_mem.c`.